app/assets/javascripts/components/static/about_page.es6.jsx

```diff
@@ -2,69 +2,85 @@ class AboutPage extends React.Component {
 
   constructor(props) {
     super(props);
-    this.state = {
-      people: [
-        'daniel', 'richard', 'rocky',
-        'rachel', 'sahil', 'lancy'
-      ],
-    }
-  }
+    this.individuals = [
```

Comment:
it was already existing. and it was initially an erb file when we first started the project. i'm not sure why it was created that way, but it maybe had to do with pathing?